src/types/course/interfaces/course.interface.ts

```diff
@@ -45,6 +46,13 @@ export interface CourseFilters extends Pick<Course, 'proficiencyLevel'> {
   category: string
   subject: string
   title: string
+  page?: string | number
+}
+
+export interface CourseFiltersActions<T> {
```

Comment:
the same interface as `FindOffersFiltersActions`
Maybe try to create separate interface for both places
and not to duplicate logic

Reply:
Replaced this interface to the `course.interface.ts` and renamed it to `FiltersActions`. `CourseFiltersActions` and `FindOffersFiltersActions` were deleted and changed its names in all code.